docs/metrics.md

```diff
@@ -28,6 +28,15 @@ Read type specifies sequential or random read.
 
 Note: Both request_count and request_latencies allows grouping by gcs method type.
 
+## File cache metrics
+* **file_cache/read_bytes_count:** The cumulative number of bytes read from file 
+cache along with read type - Sequential/Random.
+* **file_cache/read_latencies:** The cumulative distribution of the file cache read 
```

Comment:
Why don't we export the read_latencies with read_type also?

Reply:
Why I didn't export it at first place ?
1. Sequential and Random read from cache handle is similar, there is no distinction on how we fetch data for seq and random for cache hit.
2. if we add by read_type also, then we will not be able to get latency distribution just for cache hit and miss because latency numbers can't be aggregated simply (i.e. the distinction with sequential and random will always be there).